kinesis/src/main/scala/akka/stream/alpakka/kinesis/KinesisSourceStage.scala

```diff
@@ -60,20 +60,21 @@ class KinesisSourceStage(shardSettings: ShardSettings, amazonKinesisAsync: => Am
       })
 
       private def awaitingShardIterator(in: (ActorRef, Any)): Unit = in match {
-        case ((_, GetShardIteratorSuccess(result))) => {
+        case (_, GetShardIteratorSuccess(result)) => {
           iterator = result.getShardIterator
           self.become(awaitingRecords)
           requestRecords(self.ref)
         }
-        case ((_, GetShardIteratorFailure(_))) => {
+        case (_, GetShardIteratorFailure(ex)) => {
           log.error("Failed to get a shard iterator for shard {}", shardId)
+          log.error(ex.getMessage)
```

Comment:
Instead of adding an additional logging statement, we should pass in the exception to the above statement instead:

```scala
log.error(ex, "Failed ...", shardId)
```